guides/v2.3/mtf/mtf_installation.md

```diff
@@ -73,4 +77,7 @@ Find the `mtf` directory.
 Open `<magento2_root_dir>/dev/tests/functional/vendor/magento/mtf/CHANGELOG.md`. The latest version in `CHANGELOG.md` is version of the FTF you installed.
 
 ## Next Steps {#mtf_install_next} 
```

Comment:
Also, check your case per section. This one uses camel case, but a previous section uses sentence case. Be consistent, for all versions in this PR.

Reply:
Go it.